integration-tests/cypress.config.js

```diff
@@ -0,0 +1,29 @@
+const { defineConfig } = require('cypress');
+
+module.exports = defineConfig({
+  viewportWidth: 1920,
+  viewportHeight: 1080,
+  screenshotsFolder: './integration-tests/screenshots',
+  videosFolder: './integration-tests/videos',
+  video: false,
+  reporter: '../../node_modules/cypress-multi-reporters',
+  reporterOptions: {
+    configFile: 'reporter-config.json',
+  },
+  fixturesFolder: 'fixtures',
+  defaultCommandTimeout: 30000,
+  retries: {
+    runMode: 1,
+    openMode: 0,
+  },
+  e2e: {
+    // We've imported your old cypress plugins here.
+    // You may want to clean this up later by importing these.
+    setupNodeEvents(on, config) {
+      return require('./plugins/index.ts')(on, config);
+    },
```

Comment:
I'm a little unclear on what this comment means, but it sounds like there is some clean up for us to do here?

Reply:
> I'm a little unclear on what this comment means, but it sounds like there is some clean up for us to do here?

@spadgett This comment was added after I updated cypress from v9 to 11. The `require` was throwing an eslint error,  which I fixed. I will remove the comment.